css/style.css

```diff
@@ -0,0 +1,140 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
+
+
+html {
+    font-size: 10px;
+    font-family: 'Open Sans', sans-serif;
+}
+
+body {
+    margin: 0;
+}
+
+.container {
+    width: 100%;
+    height: 100vh;
+    box-sizing: border-box;
+}
+
+#welcome-section {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+    font-size: 3.5rem;
+    color: rgb(209, 209, 209);
+    background: rgb(158,157,157);
+    background: linear-gradient(145deg, rgba(158,157,157,1) 0%, rgba(40,40,40,1) 47%, rgba(39,75,103,1) 69%, rgba(84,124,184,1) 100%, rgba(48,60,105,1) 100%);
+    margin: 0;
+}
+
+#projects {
+    text-align: center;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: rgb(27, 27, 27);
+}
+
+#projects h1{
+    font-size: 2.5rem;
+    color: rgb(27, 121, 63);
+}
```

Comment:
- [ ] As a suggestion, try not to style too much on an id, as it is less flexible for larger projects when you might need to apply similar attributes. Check for example there are four attributes with the same values between these two